src/supertux/gameconfig.hpp

```diff
@@ -72,6 +72,9 @@ class Config
   /** this variable is set if tux should spawn somewhere which isn't the "main" spawn point*/
   boost::optional<Vector> tux_spawn_pos;
 
+  /** The level that should be launched in the editor*/
+  boost::optional<std::string> editor_level;
```

Comment:
If we make `--edit-level` consistent with the current naming style, this could be `edit_level` (similar to `start_demo`, `play_demo`, and `start_level`).